q2_types/per_sample_sequences/_transformer.py

```diff
@@ -351,7 +360,7 @@ def _9(fmt: PairedEndFastqManifestPhred64) \
 
 
 @plugin.register_transformer
-def _12(dirfmt: SingleLanePerSampleSingleEndFastqDirFmt) \
+def _27(dirfmt: SingleLanePerSampleSingleEndFastqDirFmt) \
```

Comment:
DBC: Why not keep this as `_12`, and name the new one `_27`? Just so you know, with transformers the names of the functions don't matter, we just took the strategy of naming them `_number`, but it can be anything

Reply:
I wanted to maintain the numerical order within the file, but I can do that

Reply:
Honestly, probably just because I saw `_10`, `_11` and was bothered at the idea of the next thing being `_27` but I guess the rest isn't in numerical order anyway :laughing: